CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.5.6
+
+* Update tax and benefit system initialization method in scripts.
```

Comment:
> A quick look at the commit
Well, the whole point is that we release a package so people don’t have to clone repos and remember to update them by themselves  :wink:

I’m just saying: copy-pasting a commit message could be useful for a contributor, but there's a different readership with consumers.
I’m sorry, but even looking at the diff I can’t tell what’s the impact of the changes on my parser-dependant code. I need the changelog to tell me if I need to do anything, and whether it is worth updating my own dependencies. It’s not the same if it says “Add compatibility with OpenFisca-France v1.2 API.” or if it says “Update method x in file y”. The later is indeed redundant with the diff and brings no value, but the former seems to me to be the very least needed to allow users to depend on open-source packages.